test/hotspot/gtest/unittest.hpp

```diff
@@ -28,7 +28,10 @@
 #include <stdio.h>
 
 #define GTEST_DONT_DEFINE_TEST 1
+
+#include "utilities/vmassert_uninstall.hpp"
 #include "gtest/gtest.h"
+#include "utilities/vmassert_reinstall.hpp"
 
 // gtest/gtest.h includes assert.h which will define the assert macro, but hotspot has its
```

Comment:
This was removed in the 17u version. Why not here as well?

Reply:
Updated in new commit.